server/backend/linux.py

```
@@ -1 +1,608 @@
+        usage=usage,
+        incomplete_details=incomplete_details,  # pyright: ignore[reportArgumentType]
+    )
+    if metrics:
+        try:
+            resp.metadata["metrics"] = metrics
+        except Exception:
+            pass
+    try:
+        _responses[response_id] = resp
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)